test/v3/fixtures/AmendHistoricNonFhlUkPiePeriodSummary/AmendHistoricNonFhlUkPiePeriodSummaryFixtures.scala

```diff
@@ -17,11 +17,7 @@
 package v3.fixtures.AmendHistoricNonFhlUkPiePeriodSummary
 
 import play.api.libs.json.{JsValue, Json}
-import v3.models.request.amendHistoricNonFhlUkPiePeriodSummary.{
-  AmendHistoricNonFhlUkPiePeriodSummaryRequestBody,
```

Comment:
I assume `AmendHistoricNonFhlUkPiePeriodSummaryRequestBody` is the same as `AmendHistoricNonFhlUkPeriodSummaryRequestBody`. There seemed to be a discrepancy between the naming of these (or at least the directory it used to be in).